enrichments/trace/internal/elastic/attributes.go

```diff
@@ -47,4 +47,15 @@ const (
 	AttributeSpanDestinationServiceResource = "span.destination.service.resource"
 	AttributeSpanDurationUs                 = "span.duration.us"
 	AttributeSpanRepresentativeCount        = "span.representative_count"
+
+	// span event attributes
+	AttributeParentID              = "parent.id"
+	AttributeErrorID               = "error.id"
+	AttributeErrorExceptionType    = "error.exception.type"
+	AttributeErrorExceptionMessage = "error.exception.message"
+	AttributeErrorExceptionHandled = "error.exception.handled"
+	AttributeErrorStacktrace       = "error.stack_trace"
+	AttributeErrorGroupingKey      = "error.grouping_key"
+	AttributeEventKind             = "event.kind"
+	AttributeMessage               = "message"
```

Comment:
This can be problematic as it conflicts with the [`message` -> `body_text` alias field](https://github.com/elastic/elasticsearch/blob/7b7dd91f6208251a830b28f2fb24c3ac91414fc1/x-pack/plugin/otel-data/src/main/resources/component-templates/logs-otel%40mappings.yaml#L26-L30).

Reply:
I don't think we even need this field - this is really the equivalent of `body_text`, isn't it? 

So, 1) it's either an [exception on span](https://opentelemetry.io/docs/specs/semconv/exceptions/exceptions-spans/) - I don't think there is any way to populate this field (top level message) via that API (it'll have exception.message in the attribute, but not on top level), 

or 

2) it's an [exception in logs](https://opentelemetry.io/docs/specs/semconv/exceptions/exceptions-logs/). That doc calls out that:
> Exceptions SHOULD be recorded as attributes on the [LogRecord](https://github.com/open-telemetry/opentelemetry-specification/tree/v1.35.0/specification/logs/data-model.md#log-and-event-record-definition)

and LogRecord has the body as a top level field - so if this is a log enriched with exception info, then we already have the field to store the message.

I mean, ultimately  both cases end up being a LogRecord, which then has the `body` field.

Reply:
Removed `message` in https://github.com/elastic/opentelemetry-lib/pull/99/commits/4361255189c752b09bf2bed60016060031fdf858